src/Microsoft.Diagnostics.Monitoring.WebApi/ProfilerMessage.cs

```diff
@@ -5,42 +5,47 @@
 using System.Text;
 using System.Text.Json;
 
+
+#if STARTUPHOOK || HOSTINGSTARTUP
```

Comment:
NOTE: This namespace adjustment is a result of `Microsoft.Diagnostics.Monitoring.TestCommon` having a dependency on `Microsoft.Diagnostics.Monitoring.WebApi` meaning tests that have nothing to do with the web api / models are still having it pulled in.

I briefly looked into removing that dependency but it'd cause several additional changes and so that should be a separate effort.

Reply:
Have you considered the approach of refactoring public types to a specific assembly?

Reply:
I have, unfortunately we aren't very consistent with where types live so things are pretty spread out between assemblies currently. I'd prefer that any refactoring around where types live be a separate, more conscious, effort to ensure it makes sense. We aren't locked into any of the current assemblies where public types are in -- we can refactor whenever we like without issue